AssetInformationApi/V1/Boundary/Request/Validation/AddAssetRequestValidator.cs

```diff
@@ -12,8 +12,10 @@ public AddAssetRequestValidator()
             RuleFor(x => x.AssetAddress).NotNull();
             RuleFor(x => x.AssetAddress.AddressLine1).NotNull()
                                  .NotEmpty();
-            RuleFor(x => x.AssetAddress.PostCode).NotNull()
-                                 .NotEmpty();
+            RuleFor(x => x.AssetAddress.PostCode)
+                .NotNull()
+                .NotEmpty()
+                .When(x => x.AssetManagement?.IsTemporaryAccomodation != true);
```

Comment:
Would be easier to read is it was:
`!x.AssetManagement?.IsTemporaryAccomodation`

Reply:
Nevermind, I'm guessing it's written this way because it's nullable.